framework/aster-frame/src/arch/x86/mod.rs

```diff
@@ -99,3 +99,59 @@ fn enable_common_cpu_features() {
         });
     }
 }
+
+/// Copies `count * size_of::<T>()` bytes from `src` to `dst`.
+/// The source and destination may overlap.
+///
+/// If the source and destination will never overlap, `copy_nonoverlapping` can be used instead.
+///
+/// # Safety
+///
+/// The safety requirements of this function are consistent with `core::ptr::copy`.
+#[inline]
+pub unsafe fn copy<T>(src: *const T, dst: *mut T, count: usize) {
```

Comment:
I think it is probably a good idea to name this method `fast_copy` and the other one `fast_copy_nonoverlapping` to highlight why we are implementing and using them over `ptr::copy`.

Reply:
OK, I will  change the name